internal/manifest/oci_index.go

```diff
@@ -57,6 +57,7 @@ func (index *OCI1IndexPublic) Instance(instanceDigest digest.Digest) (ListUpdate
 				Digest:    manifest.Digest,
 				Size:      manifest.Size,
 				MediaType: manifest.MediaType,
+				Platform:  *manifest.Platform,
```

Comment:
The platform can be missing (`nil`) in OCI; so the new filed must be able to express that (i.e. also be a `*….Platform`)

Reply:
Changed field to `Platform  *imgspecv1.Platform` , that should support `nil`